Doc/library/sqlite3.rst

```diff
@@ -1316,31 +1317,13 @@ Connection objects
 
    .. attribute:: row_factory
 
-      A callable that accepts two arguments,
-      a :class:`Cursor` object and the raw row results as a :class:`tuple`,
-      and returns a custom object representing an SQLite row.
-
-      Example:
-
-      .. doctest::
-
-         >>> def dict_factory(cursor, row):
-         ...     col_names = [col[0] for col in cursor.description]
-         ...     return {key: value for key, value in zip(col_names, row)}
-         >>> con = sqlite3.connect(":memory:")
-         >>> con.row_factory = dict_factory
-         >>> for row in con.execute("SELECT 1 AS a, 2 AS b"):
-         ...     print(row)
-         {'a': 1, 'b': 2}
-
-      If returning a tuple doesn't suffice and you want name-based access to
-      columns, you should consider setting :attr:`row_factory` to the
-      highly optimized :class:`sqlite3.Row` type. :class:`Row` provides both
-      index-based and case-insensitive name-based access to columns with almost no
-      memory overhead. It will probably be better than your own custom
-      dictionary-based approach or even a db_row based solution.
+      Define the default :attr:`Cursor.row_factory`
+      for cursors created off of this connection.
+      Assigning to this attribute does not affect the :attr:`!row_factory`
+      of open cursors belonging to this connection.
+      Is ``None`` by default.
```

Comment:
At least to me it feels a bit odd to mention the default value, but not what the default value actually does. But I'm not totally sure what to do about it—omit it, leave it, briefly mention the default behavior, or something else. Thoughts?

Reply:
I think expanding it slightly is the best option. Perhaps something like _"Is None by default, which implies no row factory is set", but I'm not particularly keen on that one either.

Reply:
```suggestion
      Is ``None`` by default,
      meaning a fetched row is returned as a :class:`tuple`.
```

Reply:
```suggestion
      Is ``None`` by default.
      meaning each row is returned as a :class:`tuple`.
```

Sounds good, though IMO, it's simpler to omit the "fetched" since that's a detail of its usage/operation.